x-pack/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -426,6 +453,74 @@ class IndexActionsContextMenuUi extends Component {
       </EuiOverlayMask>
     );
   };
+  renderConfirmFreezeModal = () => {
+    const oneIndexSelected = this.oneIndexSelected();
+    const entity = this.getEntity(oneIndexSelected);
+    const { freezeIndices, indexNames, intl } = this.props;
+    return (
+      <EuiOverlayMask>
+        <EuiConfirmModal
+          title={
+            intl.formatMessage({
+              id: 'xpack.idxMgmt.indexActionsMenu.freezeEntity.confirmModal.modalTitle',
+              defaultMessage: 'Confirm Freeze {entity}',
```

Comment:
it would be nice if we follow [EUI's single and bulk actions modal guidelines](https://elastic.github.io/eui/#/guidelines/modals). we use this pattern in other areas, remote cluster for example:

![image](https://user-images.githubusercontent.com/1965714/51280726-416d1480-1995-11e9-9c49-82624b4927dc.png)

![image](https://user-images.githubusercontent.com/1965714/51280707-3914d980-1995-11e9-8e82-83c164b2878f.png)

however, this freeze confirmation modal matches the other index management ones (force merge, delete), it will be more work to update all of them, so I'm okay with this being deferred 🙂

Reply:
Yeah I think that's a separate PR, definitely should do this but not as part of this one I think.